### PR TITLE
fix: update exports in package.json

### PR DESCRIPTION
**Pull request by gabrielelpidio**
## Summary

Solves: #2 

## Test plan

Tested with `pnpm link --global`:

Before:

image
image

After:
image
image

### CODE REVIEW

**Comment by ImRLopezAG** [comment]
hope they fix it soon.

**Review by gregberge — APPROVED** [review]

**Comment by gregberge** [comment]
I was hesitating with that, not sure what was using it.

File: package.json [hunk 1]
```diff
@@ -3,8 +3,13 @@
   "version": "1.0.0",
   "description": "Create reusable Tailwind CSS components with React.",
   "type": "module",
-  "types": "./dist/index.d.ts",
-  "exports": "./dist/index.mjs",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "module": "./dist/index.mjs",
```
Comment (mgcrea) [inline package.json:9]: Never seen a `"module"` key there, do you know why it is used for? According to [conditional exports spec](https://nodejs.org/docs/latest-v20.x/api/packages.html#conditional-exports), valid values seem to be `node-addons`, `node`, `import` and `require`.

Comment (gregberge) [reply package.json:9]: I don't want to provide a cjs alternative. I don't know @gabrielelpidio why did you add "module" entry? Can I remove it?

**Comment by gabrielelpidio** [comment]
I based off: [SWR package.json](https://github.com/vercel/swr/blob/main/package.json) it might be safe to remove though

File: package.json (hunk 1 above)
Comment (gabrielelpidio) [reply package.json:9]: I based it off [SWR package.json](https://github.com/vercel/swr/blob/main/package.json) it might be safe to remove though

**Pull request merged**